playbooks/roles/airship-deploy-osh/templates/site/soc/software/charts/osh/openstack-compute-kit/libvirt.yaml

```diff
@@ -28,5 +28,5 @@ data:
         cinder:
           user: {{ ses_cluster_configuration.cinder.rbd_store_user }}
           keyring: {{ ses_cluster_configuration.cinder.key }}
-          secret_uuid: 76f12f7a-16a6-11e9-864e-52540033f343
+          secret_uuid: {{ libvirt_ceph_cinder_secret_uuid }}
```

Comment:
Why not something like `{{ 99999999 | random | to_uuid }}` instead of calling shell that creates a file. Or is that uuid persistent and shouldn't change while running multiple times?

Reply:
I prefer it not to change when the playbook reruns. Otherwise it causes the site manifest content to change which triggers the Airship to create a new version of the site config.

Reply:
Exactly--if a new UUID is generated and applied with each run of the playbook, even a totally unrelated change would trigger an unnecessary (and as James points out, expensive) redeployment of cinder, nova, and libvirt since the new UUID would be seen as a new configuration change for those charts.